tests/zfs-tests/tests/functional/cli_root/zpool_initialize/zpool_initialize_multiple_pools.ksh

```diff
@@ -0,0 +1,131 @@
+#!/bin/ksh -p
+# SPDX-License-Identifier: CDDL-1.0
+#
+# CDDL HEADER START
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2025 Hewlett Packard Enterprise Development LP.
+#
+
+. $STF_SUITE/include/libtest.shlib
+. $STF_SUITE/tests/functional/cli_root/zpool_initialize/zpool_initialize.kshlib
+
+#
+# DESCRIPTION:
+#	Verify 'zpool initialize -a' works correctly with multiple pools
+#
+# STRATEGY:
+#	1. Create multiple pools.
+#	2. Start a initialize operation on all pools using 'zpool initialize -a'.
+#	3. Verify that the initializing is active on all pools.
+#	4. Wait for the initialize operation to complete.
+#	5. Verify that the initialize operation is complete on all pools.
+#	6. Start a initializing on all pools using 'zpool initialize -w -a'.
+#	7. Verify that the initialize operation is complete on all pools.
+#	8. Now test the -u, -c and -s options on multiple pools with -a.
+#	9. Verify that the initialize status is correctly updated on all pools.
+#
+
+verify_runnable "global"
+
+cleanup() {
+    for pool in {1..4}; do
+        log_must zpool destroy $TESTPOOL${pool}
+        log_must rm -rf $TESTDIR${pool}
+    done
+    log_must rm -f $DISK1 $DISK2 $DISK3 $DISK4
```

Comment:
Cleanup is expected to run in any conditions.  And in this case `log_must` looks too strict and I think may fail if some of pools do not exist (yet?).  We don't want incomplete cleanup if the cleanup also fail.

Reply:
Fixed.